docs/pages/reference/networking.mdx

```diff
@@ -14,13 +14,22 @@ configuration file. The public address can take an IP or a DNS name. It can also
 be a list of values:
 
 ```yaml
-public_addr: ["proxy-one.example.com", "proxy-two.example.com"]
+public_addr: ["service1.example.com", "service2.example.com"]
 ```
 
+<Admonition
+  type="warning"
+  title="Note"
+>
+  Only a single Proxy Service `public_addr` should be configured. Attempting
+  to have multiple addresses can result in redirects to the first listed address
+  that may not be available to the client.
+</Admonition>
+
 Specifying a public address for a Teleport service may be useful in the
 following use cases:
 
-- You have multiple identical services, e.g., Proxy Service instances, behind a
+- You have multiple identical services, e.g., DB Service instances, behind a
```

Comment:
I'm not so sure about this change. DB services are almost always in reverse tunnel mode so there's rarely a good reason to use a LB.

Reply:
> I'm not so sure about this change. DB services are almost always in reverse tunnel mode so there's rarely a good reason to use a LB.

Makes sense. I restored to Proxy Service in that it does make sense to put it behind a load balancer and not violating the single address bit.